README.rst

```diff
@@ -323,7 +323,39 @@ Now you can encrypt a file using::
 
 And decrypt it using::
 
-     $ sops --decrypt test.enc.yaml
+    $ sops --decrypt test.enc.yaml
+
+
+Encrypting and decrypting from other programs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using ``sops`` in scripts or from other programs, there are often situations where you do not want to write encrypted or decrypted data to disk. The best way to avoid this is to pass data to SOPS via stdin, and to let SOPS write data to stdout. By default, the encrypt and decrypt operations write data to stdout already. To pass data via stdin, you need to pass ``/dev/stdin`` as the input filename. Please note that this only works on Unix-like operating systems such as macOS and Linux. On Windows, you have to use named pipes.
```

Comment:
I'm not sure about Windows here. Is it true that you need named pipes for this? Or is there another mechanism?

Reply:
With https://github.com/getsops/sops/pull/1104, it would simply be a `-`. I would need to revise this to incorporate my own feedback, as it appears the contributor gave up.

Reply:
I'll update this section once I have recreated the above.